packages/sherlock/src/Sherlock.js

```diff
@@ -343,22 +345,35 @@ class MapServiceProvider extends ProviderBase {
     this.setUpQueryTask(serviceUrl, options);
   }
 
-  async setUpQueryTask(serviceUrl, options) {
+  setUpQueryTask(serviceUrl, options) {
     const { Query, QueryTask } = this.modules;
     const defaultWkid = 3857;
     this.query = new Query();
     this.query.outFields = this.getOutFields(options.outFields, this.searchField, options.contextField);
     this.query.returnGeometry = false;
     this.query.outSpatialReference = { wkid: options.wkid || defaultWkid };
+    this.query.orderByFields = [this.searchField];
 
     this.queryTask = new QueryTask({ url: serviceUrl });
   }
 
-  async search(searchString) {
+  async search(searchString, maxresults) {
     console.log('sherlock.MapServiceProvider:search', arguments);
 
-    this.query.where = this.getSearchClause(searchString);
-    const featureSet = await this.queryTask.execute(this.query);
+    if (this.abortController) {
+      this.abortController.abort();
+    }
+
+    this.abortController = new AbortController();
```

Comment:
**thought**: i'm going to assume that our compatibility matrix is good enough that we don't need to worry about this class not existing right?

Reply:
Yes, IE is the only one left behind on this. From my perspective, we no longer support IE in new applications.